toolkits/landcover/impl/composites.js

```diff
@@ -81,41 +81,45 @@ function createTemporalComposites(
 }
 
 /**
- * Computes a medioid composite by finding the median of one band
- * and returning the complete observation where that band is closest
- * to the median value.
+ * Computes a medioid composite by finding the median and returning the
+ * complete observation closest to the median value as measured by
+ * euclidiean distance over the specified band(s).
  *
  * @param {!ee.ImageCollection} collection The collection to composite.
- * @param {string|number} indexBand The name of the band which to
- *    select the median value. If omitted, the first band is used.
+ * @param {Array<string|number>|string|number} bands A band identifier
+ *    (a name, index or regexp) or a list of identifiers of the bands
+ *    to use to compute the median value. If omitted, all bands are used.
  * @return {!ee.ImageCollection}
  */
-function createMedioidComposite(collection, indexBand) {
-  // Compute the median of the index band.
-  indexBand = indexBand === undefined ? 0 : indexBand;
-  var median = collection.select(indexBand).median();
-  
-  // Add a band containing the difference between the index band value for each
-  // pixel and the median of the index band across all pixels in the collection.
-  var mosaic = collection.map(function(img) {
-    var diff = median.subtract(img.select(indexBand)).abs();
-    var index = ee.Image(0).subtract(diff);
-    return img.addBands(index.rename('medioid_index_'));
-  }).qualityMosaic('medioid_index_');
+function createMedioidComposite(collection, bands) {
+  bands = bands === undefined ? ".*" : bands;
```

Comment:
Nit: Let's use `'` string delimiter for consistency.

Reply:
Also, `=== undefined` and ternary aren't necessary - you can simply do;
`bands = bands || '.*'`

Reply:
Done

Reply:
Not true.
Bands can be 0 and the || would result in selecting all bands instead of band 0.